kb/temporal-platform-limits-sheet.md

```diff
@@ -45,9 +45,11 @@ Hard limits fail with an error; soft limits produce a warning log on the server
     - `limit.numPendingSignals.error`
     - `limit.numPendingCancelRequests.error`
     - `limit.numPendingChildExecutions.error`
-- [Search Attributes maximums](/visibility/#search-attributes-maximums)
+- [Search Attributes maximums](visibility/#custom-search-attributes-limits)
```

Comment:
```suggestion
- [Search Attributes maximums](/visibility/#custom-search-attributes-limits)
```
Without the leading slash, the link renders as `/kb/visibility/#custom-search-attributes-limits`.